src/required_resources.h

```diff
@@ -98,6 +97,10 @@ class RequiredResources {
   std::vector<std::string> cgroup_path_args;
   std::vector<SizedType> non_map_print_args;
   std::vector<std::tuple<std::string, long>> skboutput_args_;
+  // While max fmtstring args size is not used at runtime, the calculation
+  // is tricky enough that we want to minimize repetition in the codebase.
+  // So keep it here rather than in CodegenResources.
```

Comment:
Oh, I meant the tuple creation. Cuz each arg in the format string needs struct alignment on the stack so that writing values to them is aligned. I'll update the comment to make that more clear